components/Match/Match.css

```diff
@@ -0,0 +1,34 @@
+.Header{
+	text-align: center;
+	display: block;
+	text-transform: uppercase;
+	margin-top: 0;
+    margin-bottom: 30px;
+}
+
+.DireWin {
```

Comment:
I think we should just have a predefined dire/radiant color in palette.css and use it for all Radiant/Dire and win/loss context classes